rotkehlchen/db/settings.py

```diff
@@ -54,6 +62,8 @@ class DBSettings(NamedTuple):
     btc_derivation_gap_limit: int = DEFAULT_BTC_DERIVATION_GAP_LIMIT
     calculate_past_cost_basis: bool = DEFAULT_CALCULATE_PAST_COST_BASIS
     display_date_in_localtime: bool = DEFAULT_DISPLAY_DATE_IN_LOCALTIME
+    current_price_oracles: List[str] = DEFAULT_CURRENT_PRICE_ORACLES
```

Comment:
If you follow what I said in encoding these would become a list of enum values. And you would have to serialize/deserialize before writing/reading from the DB. Do that.